scripts/providers/aws/cloud-config.yaml

```diff
@@ -28,8 +28,9 @@ write_files:
 
       [Service]
       Type=simple
-      ExecStart=/usr/bin/meilisearch --db-path /var/lib/meilisearch/data.ms --env development --dumps-dir MEILI_DUMPS_DIR
       Environment="MEILI_SERVER_PROVIDER=aws"
+      Environment="MEILI_DUMP_DIR=/var/opt/meilisearch/dumps"
```

Comment:
Why you sometimes set `Environment="MEILI_DUMP_DIR=/var/opt/meilisearch/dumps"` and other places have `Environment="MEILI_DUMP_DIR=$MEILI_DUMP_DIR"`

Reply:
This is a really good question! `systemd` doesn't read the environment variable, so that is why there are defined directly on the file. It should probably be better to use `EnvironmentFile` option and create an env file I didn't take time to do it but I maybe will before v1 if I can.